experimental/dds/tree2/src/feature-libraries/modular-schema/modularChangeFamily.ts

```diff
@@ -342,6 +356,11 @@ export class ModularChangeFamily
 				  ).reverse(),
 			change.change.constraintViolationCount,
 		);
+
+		if (cache !== false) {
```

Comment:
```suggestion
		if (cache) {
```

Reply:
the lint is complining with this message if I put it as you sugest:
Unexpected nullable boolean value in conditional. Please handle the nullish case explicitly.eslint@typescript-eslint/strict-boolean-expressions

Reply:
Oh yea, then just do `cache === true` instead.